common-libs/internal-api-clients/src/helpers/DidAuthAdapter.ts

```diff
@@ -1,8 +1,9 @@
-import { DidAuthService } from '@affinidi/affinidi-did-auth-lib'
+import { DidAuthService, LocalExpiringDidAuthResponseToken } from '@affinidi/affinidi-did-auth-lib'
 import { AffinidiDidAuthServiceOptions } from '@affinidi/affinidi-did-auth-lib/dist/DidAuthService/DidAuthService'
 
 export class DidAuthAdapter {
   private readonly _didAuthService: DidAuthService
+  public tokenRequestTime!: number
```

Comment:
I _really_ don't like the idea of storing some state in this service. Especially one that is modifiable by external code (public non-readonly field).

`DidAuthSession` seems to be a more proper place for that.

Reply:
(because it corresponds to an actual session, and it stores the response token, so it makes sense to store a request time there as well)